sdk/storage/azdatalake/file/responses.go

```diff
@@ -73,6 +73,9 @@ type DownloadResponse struct {
 	// AcceptRanges contains the information returned from the Accept-Ranges header response.
 	AcceptRanges *string
 
+	// AccessControlList contains the combined list of access that are set for user, group and other on the file
+	AccessControlList *string
```

Comment:
Can you also add a test for checking acl in DownloadResponse

Reply:
Added